frontend/benefit/applicant/src/hooks/useUserQuery.ts

```diff
@@ -41,7 +51,7 @@ const useUserQuery = (
     () => handleResponse<UserData>(axios.get(BackendEndpoint.USER_ME)),
     {
       refetchInterval: FIVE_MINUTES,
-      enabled: !logout,
+      enabled: isEnabled(),
```

Comment:
Not sure about the side-effects of this, namely the calling for `/me` throughout the apps. Should be fine but let's keep this in mind if app halts to an error fiesta.

Reply:
If the `/me` call fails, the app redirects to the login page. Accessibility statement needs to be seen by unauthenticated users too. This logic could be transferred to the `handleError` function, though.